### DISCUSSION TITLE
Zephyrus M16 (i9 13900H, Rtx 4070, Ram 16GB, GU604VI ) keyboard heating

### DISCUSSION THREAD

**Discussion by gariseverus** [seerge/g-helper#2738] [General]
The machine is amazing from display to performance but In demanding games, the keyboard gets so hot that my fingers are burning the hottest point is in the middle.  how to cool the system What settings of G-helper to put? the temperature of the GPU & CPU does not exceed 80 degrees there is no heat...  Please help... I will be very glad for your advice

**Comment by seerge** [comment]
@gariseverus hello,

You can try to set CPU power limits, like 40W or so (under ``Fans + Power`` and click ``Apply``). That can help to tame heat a bit.

And also set a bit higher fan speeds for high temps as well. But other than that - it's just physics, device consumes a lot of power and produces heat :|

**Reply by gariseverus** [reply]
Thanks for the answer, I’ll try it, I was advised to undervolting, I don’t understand it, I’ll write an answer in a maximum of 2 days..

**Reply by seerge** [reply]
You can't undervolt Intel 13900H. Even if you would be able to - you would  get something like couple of percentage points in performance only.

**Reply by OmerPNW** [reply]
@seerge Is it possible to underclock/undervolt my GPU rtx 4080 (i9 13900h cpu i wont undervolt based on what u said + what i read elsewhere) using msi afterburner while running g helper ? Like is there any chance it might affect with the voltage vs frequency profile ?

**Reply by OmerPNW** [reply]
> You can't undervolt Intel 13900H. Even if you would be able to - you would get something like couple of percentage points in performance only.

for some reason, I am unable to see the gpu tab for fans+power for my rtx 4080 thats why I am wondering whether I need to use msi afterburner

**Reply by OmerPNW** [reply]
Ok nvm . Sorry for bothering. Apparently, i see those options if i plug in my laptop to the charging port. On battery the GPU tab doesnt show up

**Reply by seerge** [reply]
@OmerPNW when GPU is disabled (like in Optimizef on battery) - you obviously can’t tune it anyhow
